CHANGELOG.md

```diff
@@ -7,211 +7,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- [#10682](https://github.com/MetaMask/metamask-extension/pull/10682): Adding default properties to NetworkForm
-- [#10689](https://github.com/MetaMask/metamask-extension/pull/10689): make migration more safe
-- [#10639](https://github.com/MetaMask/metamask-extension/pull/10639): refactor incoming tx controller
-- [#10666](https://github.com/MetaMask/metamask-extension/pull/10666): Fixing incorrectly typed token decimal attribute
-- [#10627](https://github.com/MetaMask/metamask-extension/pull/10627): fix: replace dnode background with JSON-RPC
-- [#10661](https://github.com/MetaMask/metamask-extension/pull/10661): Improve specificity of `test:unit:lax` npm script
-- [#10664](https://github.com/MetaMask/metamask-extension/pull/10664): Fixing migration script generation paths
-- [#10655](https://github.com/MetaMask/metamask-extension/pull/10655): colocate tests in flat structure
-- [#10656](https://github.com/MetaMask/metamask-extension/pull/10656): Allow TextField to receive min and max attributes
-- [#10652](https://github.com/MetaMask/metamask-extension/pull/10652): Sync `master` with `develop`
-- Merge remote-tracking branch 'origin/develop' into sync-master
-- [#10614](https://github.com/MetaMask/metamask-extension/pull/10614): Move swaps constants to the shared constants directory
-- [#10640](https://github.com/MetaMask/metamask-extension/pull/10640): no more node:console
-- [#10634](https://github.com/MetaMask/metamask-extension/pull/10634): Setting balance to 0x0 when the original value is undefined
-- [#10615](https://github.com/MetaMask/metamask-extension/pull/10615): remove transactionCategory in favor of more types
-- [#10524](https://github.com/MetaMask/metamask-extension/pull/10524): Ensure permission log will only store JSON-able data
-- [#10624](https://github.com/MetaMask/metamask-extension/pull/10624): Replace logic for eth swap token in fetchQuotesAndSetQuoteState with getSwapsEthToken call
+- [#10952](https://github.com/MetaMask/metamask-extension/pull/10952): Prevent lag in external monitors when using Mac + Chrome
+- [#11127](https://github.com/MetaMask/metamask-extension/pull/11127): Fixes token removal when cancelling a swaps quote
+- [#10956](https://github.com/MetaMask/metamask-extension/pull/10956): Fixes intermittent token display rounding error, better handling network errors for token fetches.
+- [#11097](https://github.com/MetaMask/metamask-extension/pull/11097): Fixes account search results ordering
+- [#10884](https://github.com/MetaMask/metamask-extension/pull/10884): Fixes localStorage migration error in Firefox response
+- [#10762](https://github.com/MetaMask/metamask-extension/pull/10762): Ensure that priceSlippage fiat amounts are always shown in the quote view
+- [#10763](https://github.com/MetaMask/metamask-extension/pull/10763): Swaps: Ensure that the approval fee in the swaps custom gas modal is in network specific currency
```

Comment:
10763, 10753, 10743, 10744 were all in v9.5.0

Reply:
or rather, they were part of QA response for v9.3.0

Reply:
removed